packages/elements/src/interactive-chart/index.ts

```diff
@@ -745,6 +752,8 @@ export class InteractiveChart extends ResponsiveElement {
         this.rowLegend = this.shadowRoot.querySelectorAll('.row');
       }
       this.chart.subscribeCrosshairMove(this.handleCrosshairMoved);
+      // Legend relates to value of each series that relate to priceScale. But the axis is no event for now. So use x-axis instead.
```

Comment:
How this comment explain next line code.

Reply:
I try to describe why I choose to subscribe sizeChange event. When we come back, you won't be confused why subscribe at x-axis event. The event has only at x-axis (timeScale). Currently, y-axis (priceScale) doesn't have it.

Reply:
I mean, does `subscribeSizeChange` have anything to do with the axis in this comment?
Is it not confusing in the further?

Reply:
Updated to add more description to be clear.